### PR TITLE
Search Functionality

### PR DESCRIPTION
**Pull request by kart-c**
## Search functionality on the products page

- Search bar only appears on the products page 
- Show filtered products based on the input in the search input
- Show the number of products present on the page at any moment

### CODE REVIEW

File: src/Utils/search-products.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const searchProducts = (products, value) =>
+	products.filter((product) =>
+		product.title.toLowerCase().includes(value.toLowerCase()) ? product : null
+	);
```
Comment (AnkurChunekar) [inline src/Utils/search-products.js]:
```javascript
export const searchProducts = (products, value) =>
  products.filter((product) =>
    product.title.toLowerCase().includes(value.toLowerCase())
  );
```
`includes()` methods returns a boolean value, so if you want you want you can trim the fat!

**Review by AnkurChunekar — APPROVED** [review]
- Search functionality works fine.
- good job @kart-c

**Review by AtulPant2704 — APPROVED** [review]
The functionality is working fine and variable naming is also good, good job.

File: src/Utils/search-products.js (hunk 1 above)
Comment (kart-c) [reply src/Utils/search-products.js]: Thank you for pointing that out, I will remove the unnecessary code

**Pull request merged**